tests/r/FormatRBearTest.py

```diff
@@ -3,9 +3,9 @@
 
 good_file = """1 + 1
 if (TRUE) {
-    x <- 1  # inline comments
+    x = 1  # inline comments
 } else {
-    x <- 2
+    x = 2
     print("Oh no... ask the right bracket to go away!")
 }""".splitlines(keepends=True)
```

Comment:
you don't test the new configs, please add some testcases where you modify the settings :+1: